src/python/pants/rules/core/list_target_types.py

```diff
@@ -55,13 +55,9 @@ def format_field(field: Type[Field]) -> str:
         #  hints, but it might be weird to handle Optional.
         field_alias_text = f"  {field.alias} = ...,"
```

Comment:
I agree that `...,` is repetitive and doesn't add much.

 I don't love saying `compatibility = None`, though, because it's not quite obvious (to me at least) that that means a default.

I think what I'd really like us to do is to document the type that we expect, e.g.

```
compatibility = Optional[str],   some description (default: None)
```

--

For now, I'm going to stick what we use for V1 `./pants targets`, but will keep your suggestion in mind for a followup PR to improve the output of this goal. 

Thanks for the suggestion!